### PR TITLE
Fix wrong JobStatus Names

### PR DESCRIPTION
**Pull request by alexanderj1988**
BrigadeJS uses the default Kubernetes Pod Lifecycle as JobStatus
https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/

JobStatus -> https://github.com/Azure/brigade/blob/master/pkg/brigade/job.go
Worker -> https://github.com/Azure/brigade/blob/master/pkg/brigade/worker.go

Since the current implementation diverges from this names this pull request fixes this issue

regards Alex

### CODE REVIEW

File: src/app/dashboard/dashboard.component.ts [hunk 1]
```diff
@@ -1,8 +1,9 @@
...
-import { Component, OnInit } from '@angular/core';
-import { ActivatedRoute } from '@angular/router';
-import { LastBuild } from '../models/last-build';
-import { ProjectsBuild } from '../models/projects-build';
+import {Location} from '@angular/common';
+import {Component, OnInit} from '@angular/core';
+import {ActivatedRoute} from '@angular/router';
+import {LastBuild} from '../models/last-build';
+import {ProjectsBuild} from '../models/projects-build';
+import {JobStatus} from '../models/job';
```
Comment (radu-matei) [inline src/app/dashboard/dashboard.component.ts]: nit: do the first 5 lines change in any significant way?

File: src/app/models/job.ts [hunk 2]
```diff
@@ -1,3 +1,28 @@
+
+// same as in https://github.com/Azure/brigade/blob/master/pkg/brigade/job.go
+export enum JobStatus {
```
Comment (radu-matei) [inline src/app/models/job.ts]: I really like this solution!

File: src/app/dashboard/dashboard.component.ts (hunk 1 above)
Comment (alexanderj1988) [reply src/app/dashboard/dashboard.component.ts]: No, my ide seems to have collapsed the whitespaces :/

**Comment by vdice** [comment]
This change seems good to go.  Not sure what happened to the CI run but tests/e2e pass on this branch for me.  cc @radu-matei

**Pull request merged**